src/utils/helpers.js

```diff
@@ -73,3 +73,15 @@ export const flattenArray = arr =>
     (Array.isArray(item)
       ? [...acc, ...flattenArray(item)]
       : [...acc, item]), []).filter(item => !!item);
+
+/**
+ * Returns the size of a given string in bytes
+ * If the passed parameter is not a string, it returns 0
+ *
+ * @param {string} str - a random string
+ * @returns {number} - string size for strings, 0 for others
+ */
+export const sizeOfString = (str) => {
+  if (typeof str === 'string') return str.length * 2;
```

Comment:
🐛 The size is shown double the size
```suggestion
  if (typeof str === 'string') return str.length;
```

You can see it e.g. on this transaction https://testnet-explorer.lisk.io/tx/10611085484582831767

Reply:
This logic was wrong. we already had it in Send and Request components. I updated the utility and reused in those components